WTWSMS/decisions/GR_Scythian.txt

```diff
@@ -203,7 +203,7 @@ decisions = {
 	}
 }
 
-targetted_decisions = {
+decisions = {
```

Comment:
It's ridiculous, makes no sense, and effectively hides the Decision. I only discovered several of the ones that were foolishly moved by accident.